### ISSUE TITLE
Large multichan-ts page 'open app' opens a different app

### ISSUE THREAD

**Issue by droumis** [holoviz-topics/examples#455]
Just noting that clicking 'open app' in the large multichan-ts page opens a different app (the non-large app).. I think it's fine, but might confuse someone.

![Image](https://github.com/user-attachments/assets/6e5492c0-3e73-426b-8db8-ae4a1397eff7)

**Comment by maximlt** [comment]
Yeah examples don't declare which notebooks are served as Panel apps so the site just provides a link to the index of the deployment, in this case it's a specific app but I think that other multi-notebook examples (e.g. attractors) have multiple ones. It would be a nice improvement (though I don't hope we have so many multi-notebook examples).
